src/plugins/lightscript.js

```diff
@@ -143,7 +202,7 @@ pp.parseWhiteBlock = function (allowDirectives?, isExpression?) {
   // must start with colon or arrow
   if (isExpression) {
     this.expect(tt.colon);
-    if (!this.isLineBreak()) return this.parseMaybeAssign();
+    if (!this.isLineBreak()) return this.parseWhiteBlockBody(node, indentLevel, true, false);
```

Comment:
If I understand correctly, the compiler is going to turn that into a ternary, and the break will get wrapped into an iife by `replaceExpressionWithStatements` which I think might be illegal JS. Will test and see what happens.